src/lisflood/hydrological_modules/soilloop_merge2.py

```diff
@@ -0,0 +1,1225 @@
+"""
```

Comment:
The following scripts:
- soilloop_merge2.py
- soilloop_imposeMinLoops.py
- soilloop_alwaysMaxNoLoops.py
- soilloop_OLDvarstruct.py
are NOT required in the master branch.

The scripts listed above allow to keep track of the steps and of the investigations that were done to edit the soilloop. They might be useful in the future for other analysis but at this stage they are not required to run the code.